tests/Http/HttpRequestTest.php

```diff
@@ -315,8 +315,6 @@ public function testJSONEmulatingPHPBuiltInServer()
     {
         $payload = ['name' => 'taylor'];
         $content = json_encode($payload);
-        // The built in PHP 5.4 webserver incorrectly provides HTTP_CONTENT_TYPE and HTTP_CONTENT_LENGTH,
-        // rather than CONTENT_TYPE and CONTENT_LENGTH
```

Comment:
Out of interest: why did you remove these, Graham? I find that comments like this help understand why certain strange test cases are added...